log_drains/list.go

```diff
@@ -0,0 +1,36 @@
+package log_drains
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+)
+
+func List(app string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	logDrains, err := c.LogDrainsList(app)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	t := tablewriter.NewWriter(os.Stdout)
+
+	t.SetHeader([]string{"appID", "URL"})
```

Comment:
```suggestion
	t.SetHeader([]string{"URL"})
```

Not sure it's useful to display the app ID here

The Notion section "New CLI Commands" displays an example of wanted output. I think only the URL can currently be displayed.